packages/matter.js/src/behavior/definitions/descriptor/DescriptorServer.ts

```diff
@@ -48,23 +52,76 @@ export class DescriptorServer extends DescriptorBehavior {
                     revision: partType.deviceRevision,
                 },
             ];
+            for (
```

Comment:
See discussion in Discord ... it is unclear which consequences this has based on specs and "duplicate condition" for device types.
So I would propose to comment it out ( so that it is easy to readd based on the clarification on spec level ... for now lets just allow a dev to add it manually

Reply:
OK